File: Medium/New Year Chaos/minimumBribes.py
def minimumBribes(q):
    moves = 0

    for pos, val in enumerate(q):
        if (val-1) - pos > 2:
            print("Too chaotic")
            return
        
        for j in range(max(0,val-2), pos):
            if q[j] > val:
                moves+=1
        
    print(moves)
    return

File: Medium/New Year Chaos/minimumBribes.js
function minimumBribes(q) {
    let moves = 0

    for (let i = 0; i < q.length; i++) {
        if ((q[i] - 1) - i > 2) {
            console.log('Too chaotic')
            return
        }

        for (let j = Math.max(0, q[i] - 2); j < i; j++) {
            if (q[j] > q[i]) {
                moves += 1
            }
        }
    }

    console.log(moves)
    return
}